src/custom/OptimizationInterface.ts

```diff
@@ -57,4 +57,8 @@ export type OptimizationResults = {
 	connectionsLightsMap: Map<number, TrafficLight[]>;
 	roadsLightsMap: Map<number, TrafficLight[]>;
 	lightsDirectionMap: Map<number, TrafficLightDirection>;
+	connectionsFlowMap: Map<number, number>;
```

Comment:
Ze wszystkich tych nazw (w całym typie) wywaliłbym słowo Map. Typ nie dodaje tutaj żadnej wartości do opisu, my wiemy czym to jest, a przed klientem i tak jest to ukrywane